recipes/cocoyaxi/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class CoConan(ConanFile):
```

Comment:
```suggestion
class CocoyaxiConan(ConanFile):
```
Please rename class name too.